modules/ossm-migrating-network-policies-security.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-network-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-network-policies-security_{context}"]
+= Migrating network policies for security reasons
+
+This procedure applies only if you must recreate your network policies first for security reasons.
+
+.Prerequisites
+
+* You have deployed {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+
+[IMPORTANT]
+====
+The `maistra.io/member-of: istio-system` label will be removed from the namespaces during migration.
+====
+
+.Procedure
+
+[IMPORTANT]
+====
+During the recreation of network policies from {SMProduct} 2 to {SMProduct} 3, both control planes must have access to all workloads and vice versa.
+====
+
+. Label namespaces.
++
+[NOTE]
+====
+Use a label scoped specifically to your mesh that can be reused for discovery selectors.
+====
```

Comment:
Are we missing an example here? The rest of the yamls use `service-mesh: enabled` so we can mention that as an example. If you wanted to include a command you can use:
```
oc label namespace <app-namespace> service-mesh=enabled`
```